GitUI/Translation/English.xlf

```diff
@@ -8271,6 +8271,18 @@ If this is a central repository (bare repository without a working directory):
         <source>Rebase current branch on</source>
         <target />
       </trans-unit>
+      <trans-unit id="rebaseToolStripMenuItem.Text">
```

Comment:
This is not necessary
This file gets generated from the code

Reply:
You do not see the new strings without this...